src/test/java/io/appium/java_client/ios/iOSGestureTest.java

```diff
@@ -134,7 +134,7 @@ public void TapSingleFingerTest() {
 
   @Test
```

Comment:
@TikhomirovSergey https://github.com/appium/appium-ios-driver/blob/master/lib/driver.js#L75

This is removed from ios driver as well.Please look into this.Correct me if i am wrong.